<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HWQwertyInput</class>
 <widget class="QFrame" name="HWQwertyInput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QPushButton" name="bQ">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>35</width>
     <height>88</height>
    </rect>
   </property>
   <property name="text">
    <string>Q</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bW">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>0</y>
     <width>35</width>
     <height>88</height>
    </rect>
   </property>
   <property name="text">
    <string>W</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bR">
   <property name="geometry">
    <rect>
     <x>108</x>
     <y>0</y>
     <width>35</width>
     <height>88</height>
    </rect>
   </property>
   <property name="text">
    <string>R</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bT">
   <property name="geometry">
    <rect>
     <x>144</x>
     <y>0</y>
     <width>35</width>
     <height>88</height>
    </rect>
   </property>
   <property name="text">
    <string>T</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bI">
   <property name="geometry">
    <rect>
     <x>252</x>
     <y>0</y>
     <width>35</width>
     <height>88</height>
    </rect>
   </property>
   <property name="text">
    <string>I</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bY">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>0</y>
     <width>35</width>
     <height>88</height>
    </rect>
   </property>
   <property name="text">
    <string>Y</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bU">
   <property name="geometry">
    <rect>
     <x>216</x>
     <y>0</y>
     <width>35</width>
     <height>88</height>
    </rect>
   </property>
   <property name="text">
    <string>U</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bP">
   <property name="geometry">
    <rect>
     <x>324</x>
     <y>0</y>
     <width>35</width>
     <height>88</height>
    </rect>
   </property>
   <property name="text">
    <string>P</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bO">
   <property name="geometry">
    <rect>
     <x>288</x>
     <y>0</y>
     <width>35</width>
     <height>88</height>
    </rect>
   </property>
   <property name="text">
    <string>O</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bA">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>35</width>
     <height>88</height>
    </rect>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bF">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>35</width>
     <height>88</height>
    </rect>
   </property>
   <property name="text">
    <string>F</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bS">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>35</width>
     <height>88</height>
    </rect>
   </property>
   <property name="text">
    <string>S</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bG">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>35</width>
     <height>88</height>
    </rect>
   </property>
   <property name="text">
    <string>G</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bD">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>35</width>
     <height>88</height>
    </rect>
   </property>
   <property name="text">
    <string>D</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bJ">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>90</y>
     <width>35</width>
     <height>88</height>
    </rect>
   </property>
   <property name="text">
    <string>J</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bK">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>35</width>
     <height>88</height>
    </rect>
   </property>
   <property name="text">
    <string>K</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bH">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>35</width>
     <height>88</height>
    </rect>
   </property>
   <property name="text">
    <string>H</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bZ">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>35</width>
     <height>88</height>
    </rect>
   </property>
   <property name="text">
    <string>Z</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bB">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>35</width>
     <height>88</height>
    </rect>
   </property>
   <property name="text">
    <string>B</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bC">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>35</width>
     <height>88</height>
    </rect>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bV">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>180</y>
     <width>35</width>
     <height>88</height>
    </rect>
   </property>
   <property name="text">
    <string>V</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bN">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>180</y>
     <width>35</width>
     <height>88</height>
    </rect>
   </property>
   <property name="text">
    <string>N</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bL">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>90</y>
     <width>35</width>
     <height>88</height>
    </rect>
   </property>
   <property name="text">
    <string>L</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bX">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>35</width>
     <height>88</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bE">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>0</y>
     <width>35</width>
     <height>88</height>
    </rect>
   </property>
   <property name="text">
    <string>E</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bM">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>180</y>
     <width>35</width>
     <height>88</height>
    </rect>
   </property>
   <property name="text">
    <string>M</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bBackspace">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>180</y>
     <width>41</width>
     <height>88</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bEnter">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>270</y>
     <width>81</width>
     <height>88</height>
    </rect>
   </property>
   <property name="text">
    <string>ENTER</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bShift">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>31</width>
     <height>88</height>
    </rect>
   </property>
   <property name="text">
    <string>SH</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bDot">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>270</y>
     <width>35</width>
     <height>88</height>
    </rect>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bSpace">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>114</width>
     <height>88</height>
    </rect>
   </property>
   <property name="text">
    <string>SPACE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bSlash">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>270</y>
     <width>35</width>
     <height>88</height>
    </rect>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b123">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>40</width>
     <height>88</height>
    </rect>
   </property>
   <property name="text">
    <string>123</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
